### ISSUE TITLE
JS Error on checkout page

### ISSUE THREAD

**Issue by ryanfern86goa** [PayU-EMEA/plugin_magento_23#14]
Hi 

We have updated the module from 1.0.4  to 1.1.2 today and now seeing this error in console.

`Uncaught ReferenceError: Unable to process binding "css: function(){return {'_active':isPayuSelected()} }"
Message: isPayuSelected is not defined
    at css (eval at createBindingsStringEvaluator (knockout.js:2982), <anonymous>:3:74)
    at update (knockout.js:4185)
    at ko.dependentObservable.disposeWhenNodeIsRemoved (knockout.js:3373)
    at Function.evaluateImmediate_CallReadThenEndDependencyDetection (knockout.js:2173)
    at Function.evaluateImmediate_CallReadWithDependencyDetection (knockout.js:2140)
    at Function.evaluateImmediate (knockout.js:2101)
    at Object.ko.computed.ko.dependentObservable (knockout.js:1954)
    at knockout.js:3371
    at Object.arrayForEach (knockout.js:159)
    at applyBindingsToNodeInternal (knockout.js:3343)`

Analysing the component in question (PayU_PaymentGateway/js/view/payment/method-renderer/payu_gateway), I can see there is no method "isPayuSelected" defined. May be you are supposed to refer to "$parent.isPayuSelected()" in template view/frontend/web/template/payment/payu_gateway.html

Please see attached screenshot for reference.
<img width="788" alt="Screenshot 2020-09-02 at 10 56 45" src="https://user-images.githubusercontent.com/5849809/91968222-3acb1680-ed0c-11ea-9f41-e26ea1371d81.png">

**Comment by regdos** [comment]
Method "isPayuSelected" has been removed from js and template in commit https://github.com/PayU-EMEA/plugin_magento_23/commit/35f4e9d6df94373074845ce719cb054d21d5e1e3

Have you refreshed the cache?

**Comment by ryanfern86goa** [comment]
Hi @regdos 

thanks fo replying so swiftly.

Yes, i did remove pub and processed dir.

Let me try again

**Comment by ryanfern86goa** [comment]
@regdos 

working now. Thanks
